components/brave_rewards/resources/extension/brave_rewards/components/panel.tsx

```diff
@@ -192,11 +192,22 @@ export class Panel extends React.Component<Props, State> {
 
     const notification: RewardsExtension.Notification = notifications[currentNotification]
 
+    // Note: This declaration must match the RewardsNotificationType enum in
+    // brave/components/brave_rewards/browser/rewards_notification_service.h
+    enum RewardsNotificationType {
```

Comment:
Please move this into `brave/components/definitions/rewardsExtensions.d.ts`

Reply:
Will do!

Reply:
Seems like it's not recommended to declare an enum in a *.d.ts file, as it leads to a runtime error (which I'm seeing when I do this).

https://lukasbehal.com/2017-05-22-enums-in-declaration-files/

Am I missing something there and it's actually OK? Or is there a different way to achieve this?

Reply:
didn't had this problem yet, but you can easily convert it into type

Reply:
OK, fixed.